comment/static/js/comment.js

```diff
@@ -111,6 +120,47 @@ document.addEventListener('DOMContentLoaded', () => {
         }).join('&');
     };
 
+    let loadAnonymousModalForm = _form => {
+        let form = _form.cloneNode(true);
+        let textareaElement = form.querySelector('textarea');
+        textareaElement.hidden = true;
+        form.querySelector('button').setAttribute('data-email-required', 'false');
+        let emailElementData = `
+                    <div class="row">
+                        <div class="col-2">
+                            <label for="email">Email:</label>
+                        </div>
+                        <div class="col-10">
+                            <input id="email" class="form-control mr-2 w-100" type="email" name="email" required>
+                            <div class="error text-danger small mt-1"></div>
+                        </div>
+                    </div>`;
+        let emailElement = stringToDom(emailElementData, '.row');
+        let inputPlaceholder = form.querySelector('#inputPlaceholder');
+        inputPlaceholder.classList.remove('col-sm-9');
+        inputPlaceholder.classList.remove('col-md-10');
+        inputPlaceholder.classList.add('col-sm-8');
+        inputPlaceholder.appendChild(emailElement);
+        let buttonPlaceholder = form.querySelector('#buttonPlaceholder');
+        buttonPlaceholder.classList.remove('col-sm-3');
+        buttonPlaceholder.classList.remove('col-md-2');
+        buttonPlaceholder.classList.add('col-sm-4');
+        createAnonymousCommentModal.querySelector('.modal-body').appendChild(form);
+        showModal(createAnonymousCommentModal);
```

Comment:
I think we should add this line after the call to `showModal`. This will bring the input element for email in focus.

```js
form.querySelector('#email').focus();
```

Also, i think the ID `email` is too generic, and might clash with some other element present on the page. May we can change it to something like `DABEmail` or anything of this sort.